src/modules/catalog.js

```diff
@@ -119,7 +124,8 @@ async function createQueryParamsAndFormData(parameters) {
 
 async function addTarArchiveToFormData(parameters, formData, operation, apiKey) {
   try {
-    const { section } = parameters;
+    const { section, onMissing } = parameters;
+    const onMissingParameter = onMissing && onMissing !== 'FAIL' ? onMissing.toLowerCase() : '';
```

Comment:
I wonder if we should throw an error if `onMissing` is not one of `FAIL`, `CREATE`, or `IGNORE` 🤔 

I just tried to catch the error that comes from the API and it's not super helpful.

`Error: expected MissingRecordsStrategy`

Reply:
Good catch. I think it's a good idea to add validation. Made the changes, not sure about the error message. Let me know what you think